WordPress/Classes/ViewRelated/Media/SiteMedia/SiteMediaPickerViewController.swift

```diff
@@ -99,6 +99,12 @@ final class SiteMediaPickerViewController: UIViewController, SiteMediaCollection
 
     // MARK: - SiteMediaCollectionViewControllerDelegate
 
+    func siteMediaViewController(_ viewController: SiteMediaCollectionViewController, contextMenuFor media: Media) -> UIMenu? {
+        UIMenu(children: [UIAction(title: viewController.isSelected(media) ? Strings.deselect : Strings.select, image: UIImage(systemName: "checkmark.circle")) { [weak self] _ in
```

Comment:
Wdyt of extracting the title to a constant to make it easier to read?
```suggestion
        let title = viewController.isSelected(media) ? Strings.deselect : Strings.select
        UIMenu(children: [UIAction(title: title, image: UIImage(systemName: "checkmark.circle")) { [weak self] _ in
```